Source/WebKit/UIProcess/WebProcessProxy.h

```diff
@@ -829,6 +835,12 @@ class WebProcessProxy final : public AuxiliaryProcessProxy {
 #if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
     IPC::ScopedActiveMessageReceiveQueue<LogStream> m_logStream;
 #endif
+
+#if ENABLE(CONTENT_EXTENSIONS)
+    bool m_resourceMonitorRuleListRequestedBySomePage { false };
+    RefPtr<WebCompiledContentRuleList> m_resourceMonitorRuleListSentTo;
```

Comment:
I would just name this `m_resourceMonitorRuleList`. Or just make it `bool m_hasSentResourceMonitorRuleList`.

Reply:
Need to update the list when new list is delivered, so comparison is required. I'll change the name as suggested.